src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -190,6 +206,7 @@ private constructor(
 
   companion object {
     private val logger = LoggerFactory.getLogger(AgentChatSession::class.java)
+    private const val defaultModel = "anthropic/claude-2.0"
```

Comment:
NIT: top-level immutable properties should be uppercase (logger is mutable, DEFAULT_MODEL is not).